app/assets/stylesheets/admin/shared/icons.scss

```diff
@@ -16,6 +16,7 @@
 }
 
 .icon-email:before    { @extend .icon-envelope:before }
+.icon-resend_authorization_email:before    { @extend .icon-envelope:before }
```

Comment:
I fail to see how and where this class is used. Can't we reuse the one above and avoid the duplication?

Reply:
This is a side effect of the metaprogramming - the icon's class is based on the action being performed, in this case the action is `resend_authorization_email`. Definitely not pretty...

Reply:
Yes, I figured...

Reply:
Going to mark as resolved so we can get this out, unless we have ideas on how to make it a little better?